src/core/tracking.js

```diff
@@ -40,37 +39,43 @@ export function isDoNotTrackEnabled({
 
 export class Tracking {
   constructor({
+    _config = config,
     _isDoNotTrackEnabled = isDoNotTrackEnabled,
-    _log = log,
-    trackingEnabled,
-    trackingId,
-    trackingSendInitPageView,
   } = {}) {
     if (typeof window === 'undefined') {
-      /* istanbul ignore next */
       return;
     }
-    this._log = _log;
-    this.id = trackingId;
-    this.enabled = trackingEnabled && trackingId && !_isDoNotTrackEnabled();
+    this._log = log;
+    this.logPrefix = '[GA]'; // this gets updated below
+    this.id = _config.get('trackingId');
+
+    if (!convertBoolean(_config.get('trackingEnabled'))) {
+      this.log('Disabled because trackingEnabled was false');
+      this.enabled = false;
+    } else if (!this.id) {
+      this.log('Disabled because trackingId was empty')
+      this.enabled = false;
+    } else if (_isDoNotTrackEnabled()) {
+      this.log(oneLine`Do Not Track Enabled; Google Analytics not
+        loaded and tracking disabled`);
+      this.enabled = false;
+    } else {
+      this.log('Google Analytics is enabled');
```

Comment:
The only people who will ever see this are probably technical and those who see "Google Analytics" as something more nefarious than it is in our usage. I don't think we're implying tracking or encouraging them to opt-out, I'm just trying to give curious users who would actually hit this log output choice and save someone from filing a bug asking "how can I disable this?"